### PR TITLE
Better code, i guess

### PR DESCRIPTION
**Pull request by andriycraft**
i didnt test it yet, so do not merge

### CODE REVIEW

**Comment by andriycraft** [comment]
done, i guess

**Comment by andriycraft** [comment]
Changes:
[/kick] reason is now optional
[skins] Fixed invalid skin size even if it is valid
[general code] Fixed some bad code
[config] Kick messages are not configurable

File: changelog.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 #### 1.0.8
```
Comment (andriycraft) [inline changelog.md:1]: typo*

**Comment by VaxPex** [comment]
and the err_... make it an object

**Comment by ProtectorYT364** [comment]
what is err

**Comment by VaxPex** [comment]
tthe config

**Comment by VaxPex** [comment]
in

**Comment by andriycraft** [comment]
> and the err_... make it an object

done, i guess

**Comment by VaxPex** [comment]
fetch & pull then i will look at it

**Comment by andriycraft** [comment]
Done

**Review by VaxPex — CHANGES_REQUESTED** [review]
the skin.js

File: src/entity/Skin.js [hunk 2]
```diff
@@ -17,7 +17,8 @@ class Skin {
     static ACCEPTED_SKIN_SIZES = [
         64 * 32 * 4,
         64 * 64 * 4,
-        128 * 128 * 4
+        128 * 128 * 4,
+        4102
```
[the remark this answers is not in the document]
Comment (ProtectorYT364) [reply src/entity/Skin.js:21]: Lol

**Review by VaxPex — APPROVED** [review]

**Comment by andriycraft** [comment]
merge

**Pull request merged**